test/test-nagoya-university-conversation-corpus.rb

```diff
@@ -0,0 +1,128 @@
+class NagoyaUniversityConversationCorpusTest < Test::Unit::TestCase
+  def setup
+    @dataset = Datasets::NagoyaUniversityConversationCorpus.new
+  end
+
+  test('#each_sentences') do
+    first_sentences = @dataset.each.to_a[0].sentences.to_a
+    last_sentences = @dataset.each.to_a[-1].sentences.to_a
```

Comment:
How about defining a variable to avoid multiple parsing?

```suggestion
    records = @dataset.each.to_a
    first_sentences = records[0].sentences.to_a
    last_sentences = records[-1].sentences.to_a
```

And can we remove `.to_a` for `sentences`?

```suggestion
    records = @dataset.each.to_a
    first_sentences = records[0].sentences
    last_sentences = records[-1].sentences
```

Reply:
Thanks for pointing out.